src/VolleyManagement.Backend/VolleyManagement.Domain/GamesAggregate/Score.cs

```diff
@@ -18,12 +18,22 @@ public Score()
         /// <param name="home">Score of the home team.</param>
         /// <param name="away">Score of the away team.</param>
         /// <param name="isTechnicalDefeat">Indicating whether the technical defeat has taken place.</param>
-        public Score(byte home, byte away, bool isTechnicalDefeat = false)
+        public Score(byte home, byte away, bool isTechnicalDefeat)
         {
             Home = home;
             Away = away;
             IsTechnicalDefeat = isTechnicalDefeat;
         }
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Score"/> class that contains specified score.
+        /// </summary>
+        /// <param name="home">Score of the home team.</param>
+        /// <param name="away">Score of the away team.</param>
+        public Score(byte home, byte away)
+        {
+            Home = home;
```

Comment:
Code is duplicated

Reply:
Sonar said thay "u must use overloading insted"

Reply:
I know that) But still duplication can be reduced and Sonar will still be happy

Reply:
ok